templates/search/blocks/search_results__selected-filters.html

```diff
@@ -6,7 +6,7 @@ <h2 class="sr-only">Selected filters</h2>
     {% for field_name, selected_options in selected_filters.items %}
         {% for value, label in selected_options %}
             {# IMPORTANT: Never apply '|safe' to value or label here, as validation is not applied to most of these fields #}
-            <li class="search-results__selected-filters-item">
+            <li class="search-results__selected-filters-item" data-filter data-filter-value="{{ field_name }}" data-filter-name="{{ label }}" data-search-bucket="{{ buckets.current.label }}">
```

Comment:
```suggestion
            <li class="search-results__selected-filters-item" data-filter data-filter-value="{{ label }}" data-filter-name="{{ field_name }}" data-search-bucket="{{ buckets.current.label }}">
```

Reply:
For date filter, you might want to check `data-filter-value=` if {{label}} or {{value}} is Ok.

Reply:
Thanks @TNA-Allan you're right I had these the wrong way round - updated now.

Reply:
Hi @janehughes , as discussed for date fields the values the format is `YYYY-MM-DD` as per [DF-524-Tracking when a user selects a date filter](https://national-archives.atlassian.net/browse/DF-524)

Reply:
@TNA-Allan managed to fix whatever was going on with my local build - have made and tested the changes for the date fields now so it's ready for a final review now :) thanks